test/cells/test_cell.py

```diff
@@ -10,75 +10,83 @@
 HERE = Path(__file__).resolve().parent
 
 
-class TestCell:
-    def test_ln_positioning_and_page_breaking_for_cell(self, tmp_path):
-        doc = fpdf.FPDF(format="letter", unit="pt")
-        doc.add_page()
-
-        doc.set_font("helvetica", size=TEXT_SIZE)
-        text = (
-            "Lorem ipsum Ut nostrud irure reprehenderit anim nostrud dolore sed "
-            "ut Excepteur dolore ut sunt irure consectetur tempor eu tempor "
-            "nostrud dolore sint exercitation aliquip velit ullamco esse dolore "
-            "mollit ea sed voluptate commodo amet eiusmod incididunt Excepteur "
-            "Excepteur officia est ea dolore sed id in cillum incididunt quis ex "
-            "id aliqua ullamco reprehenderit cupidatat in quis pariatur ex et "
-            "veniam consectetur et minim minim nulla ea in quis Ut in "
-            "consectetur cillum aliquip pariatur qui quis sint reprehenderit "
-            "anim incididunt laborum dolor dolor est dolor fugiat ut officia do "
-            "dolore deserunt nulla voluptate officia mollit elit consequat ad "
-            "aliquip non nulla dolor nisi magna consectetur anim sint officia "
-            "sit tempor anim do laboris ea culpa eu veniam sed cupidatat in anim "
-            "fugiat culpa enim Ut cillum in exercitation magna nostrud aute "
-            "proident laboris est ullamco nulla occaecat nulla proident "
-            "consequat in ut labore non sit id cillum ut ea quis est ut dolore "
-            "nisi aliquip aute pariatur ullamco ut cillum Duis nisi elit sit "
-            "cupidatat do Ut aliqua irure sunt sunt proident sit aliqua in "
-            "dolore Ut in sint sunt exercitation aliquip elit velit dolor nisi "
-            ""
-        ) * 100
-
-        for i in range(20):
-            doc.cell(
-                w=72,
-                h=TEXT_SIZE * 1.5,
-                border=1,
-                ln=2,
-                txt=text[i * 100 : i * 100 + 99],
-            )
-
-        assert_pdf_equal(
-            doc, HERE / "ln_positioning_and_page_breaking_for_cell.pdf", tmp_path
+def test_ln_positioning_and_page_breaking_for_cell(tmp_path):
+    doc = fpdf.FPDF(format="letter", unit="pt")
+    doc.add_page()
+
+    doc.set_font("helvetica", size=TEXT_SIZE)
+    # pylint has to be disabled for the variable "text". Otherwise it will be marked as
```

Comment:
Thanks for the explanatory comment.

An alternative would be to increment the `min-similarity-lines` in `.pylintrc`:
https://github.com/PyFPDF/fpdf2/blob/master/.pylintrc#L7

What do you think is the best solution?

Reply:
I've as well thought about that, but increasing it to, let's say, 50 in the pylintrc file would increase;this parameter for all the files and not only the two files in question.

Or is there a way to set `min-similarity-lines` per file? Or even inline as comment? Something like `# pylint: min-similarity-lines=50`?

Reply:
All well thought out, let's keep it that way :)